### PR TITLE
cli test for polling incoming shares

### LINKED ISSUE #35264: acceptance tests for "occ incoming-shares:poll"
part of #35207
function implemented in https://github.com/owncloud/core/pull/34903

### PR DESCRIPTION
**Pull request by individual-it**
## Related Issue

- Fixes #35264

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/cliMain/cliIncomingShares.feature [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  Scenario: poll incoming share with a federation share and no file change
+    Given using server "REMOTE"
+    And user "user0" has been created with default attributes
+    And user "user0" has created folder "/shareFolder/"
+    And using server "LOCAL"
+    And user "user1" has been created with default attributes
+    And user "user1" has stored etag of element "/"
+    And user "user0" from server "REMOTE" has shared "/shareFolder" with user "user1" from server "LOCAL"
+    And user "user1" from server "LOCAL" has accepted the last pending share
```
Comment (phil-davis) [inline tests/acceptance/features/cliMain/cliIncomingShares.feature]: This seems a bit weird that a user can accept a share and at that time nothing happens to "map" the shared folder into the local user's file system. But I guess that is OK - if/when the local user first lists the contents of their root folder and/or the content of the shared folder, at that time the remote share will be checked...

**Review by phil-davis — APPROVED** [review]
I made a comment on the behaviour in one scenario, but actually it is the way the system works and IMO is OK.

**Comment by individual-it** [comment]
@dpakach please backport

**Comment by phil-davis** [comment]
Backport `stable10` #35352

**Pull request merged**